tests/metrics-server_resource_table.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python3
+
+import subprocess
+import re
+import sys
+import os
+import glob
+from collections import defaultdict
+
+try:
+    import yaml
+    YAML_AVAILABLE = True
+except ImportError:
+    print("Warning: PyYAML not available. Install with: pip install PyYAML")
+    print("Falling back to actual usage only...")
+    YAML_AVAILABLE = False
+
+# Component categorization rules - centralized mapping
+COMPONENT_RULES = {
+    'Dex + OAuth2-Proxy': {
+        'namespaces': ['auth', 'oauth2-proxy'],
+        'keywords': ['dex', 'oauth2-proxy']
+    },
+    'Cert Manager': {
+        'namespaces': ['cert-manager'],
+        'keywords': ['cert-manager']
+    },
+    'Istio': {
+        'namespaces': ['istio-system'],
+        'keywords': ['istio', 'cluster-local-gateway', 'cluster-jwks-proxy']
+    },
+    'Knative': {
+        'namespaces': ['knative-serving', 'knative-eventing'],
+        'keywords': ['knative']
+    },
+    'Katib': {
+        'keywords': ['katib']
+    },
+    'KServe': {
+        'keywords': ['kserve', 'predictor', 'models-web-app']
+    },
+    'Metadata': {
+        'keywords': ['metadata', 'mysql'],
+        'exclude': ['model-registry']
+    },
+    'Model Registry': {
+        'keywords': ['model-registry']
+    },
+    'Pipelines': {
+        'keywords': ['ml-pipeline', 'workflow-controller', 'cache-server', 'minio', 'pipeline']
+    },
+    'Spark': {
+        'keywords': ['spark']
+    },
+    'Training Operator': {
+        'keywords': ['training-operator', 'training']
+    },
+    'Jupyter': {
+        'keywords': ['jupyter', 'notebook-controller']
+    },
+    'Central Dashboard': {
+        'keywords': ['centraldashboard']
+    },
+    'Profiles + KFAM': {
+        'keywords': ['profiles', 'profile-controller']
+    },
+    'Admission Webhook': {
+        'keywords': ['admission-webhook', 'poddefaults']
+    },
+    'Volumes Web App': {
+        'keywords': ['volumes-web-app']
+    },
+    'Tensorboard': {
+        'keywords': ['tensorboard']
+    },
+    'PVC Viewer Controller': {
+        'keywords': ['pvcviewer']
+    },
+    'Storage & Experimental': {
+        'keywords': ['seaweedfs', 'ray', 'minio-tenant']
```

Comment:
What do you mean by minio-tenant?  I think you can just ignore the experimental folder for now entirely.